web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/inputfield/InputFieldResource.java

```diff
@@ -66,15 +66,9 @@
 	AuthenticationVO auth;
 
 	@POST
-	@Consumes(MediaType.MULTIPART_FORM_DATA)
+	@Consumes({ MediaType.APPLICATION_JSON })
```

Comment:
the two methods differ by @Comsumes, as supported by jersey. the incompatibility is a deficiency in the swagger client generator. so just removing the 2nd appearance should be the final alternative only. but then doing so for all occurences.

Reply:
Sry, this wasn't my intention - I'll fix it

Reply:
we could eg. introduce a dedicated path for it, as a workaround for the clientgenerator.